build/travis.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Fail on error
+set -e
+
+REL=$(dirname "$0")
+
+# Add everything, get ready for commit. But only do it if we're on
```

Comment:
Comment bug, it isn't only on master. It's also on `<#>.<#>.X` branches I think

Reply:
Ah ha! Nice catch. I just copy/pasta'd this from the `documentation` repository since it uses a similar trick.

I had actually thought about this wondering about versioned branches and how that would all work. It's actually nice that it's `master` or `number.number.X` as well. Will need to keep that in mind going forward. Might even want to adjust this to something like `v<#>.<#>.<#>`.

Frankly we're going to need to do some testing around releases and upgrades and how to adjust the CSV and package files to allow for Operator upgrades.

Will fix the comment but leave the logic alone for now. Thanks!